config/samples/agent-install.openshift.io_v1beta1_agentserviceconfig.yaml

```diff
@@ -41,3 +41,8 @@ spec:
     rootFSUrl: https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.9/4.9.0/rhcos-4.9.0-aarch64-live-rootfs.aarch64.img
     url: https://mirror.openshift.com/pub/openshift-v4/aarch64/dependencies/rhcos/4.9/4.9.0/rhcos-4.9.0-aarch64-live.aarch64.iso
     version: 49.84.202110080947-0
+  - cpuArchitecture: x86_64
+    openshiftVersion: '4.10'
+    rootFSUrl: https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest-4.10/rhcos-4.10.0-0.nightly-2021-11-29-191648-x86_64-live-rootfs.x86_64.img
+    url: https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest-4.10/rhcos-4.10.0-0.nightly-2021-11-29-191648-x86_64-live.x86_64.iso
```

Comment:
This URL is now broken.
Don't use `latest-4.10` and then a nightly file. As soon as "latest" is no longer that nightly this will fail us.

This should have been https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/4.10.0-0.nightly-2021-11-29-191648/ I think